plugins/pulp_rpm/plugins/migrations/0028_standard_storage_path.py

```diff
@@ -0,0 +1,124 @@
+from pulp.server.db import connection
+
+from pulp.plugins.migration.standard_storage_path import Migration, Plan
+
+
+def migrate(*args, **kwargs):
+    """
+    Migrate content units to use the standard storage path introduced in pulp 2.8.
+    """
+    migration = Migration()
+    migration.add(RPM())
+    migration.add(SRPM())
+    migration.add(DRPM())
+    migration.add(Distribution())
+    migration.add(YumMetadata())
+    migration.add(ISO())
+    migration()
+
+
+class Package(Plan):
+    """
+    The (base) migration plan for package units.
+    """
+
+    key_fields = (
```

Comment:
It looks like the super class ``__init__`` just over-shadows this with an instance attribute of the same name, and with the same value. Perhaps you could take it out of ``__init__``, and let each subclass define it as a class attribute like you've done here?